examples/python_mnist/grpc_request.py

```diff
@@ -0,0 +1,860 @@
+import grpc 
+import numpy as np
+import hydro_serving_grpc as hs
+from grpc import ssl_channel_credentials
+
+#            import hydro_serving_grpc as hs 
+
+channel = grpc.secure_channel("dev.k8s.hydrosphere.io", credentials=ssl_channel_credentials())
+stub = hs.PredictionServiceStub(channel) 
+model_spec = hs.ModelSpec(name="mnist2", signature_name="INFERRED_PIPELINE_SIGNATURE")
+tensor_shape = hs.TensorShapeProto(dim=[hs.TensorShapeProto.Dim(size=-1), hs.TensorShapeProto.Dim(size=28), hs.TensorShapeProto.Dim(size=28)])
+val = np.array([
```

Comment:
Use `val = np.ones((1, 28, 28), dtype=np.float64))` instead of literal. Or load example MNIST image`